sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppDestination.kt

```diff
@@ -0,0 +1,29 @@
+package co.nimblehq.sample.compose.ui
+
+import androidx.navigation.*
+
+const val KEY_ID = "id"
```

Comment:
@luongvo Shall we use `KeyId` -- PascalCase as it is the convention from the [Jetpack Compose guideline](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#kotlin-style-singletons_constants_sealed-class-and-enum-class-values-do)?

Reply:
@lydiasama Let's keep it as `SnakeCase` for now. I will create `RFC` for voting, So after we have a finalized change to `PascalCase` then we can create an issue and update it 🚀 cc @luongvo @Tuubz

Reply:
@lydiasama I think we should only use PascalCase for Jetpack Compose components. This issue still makes me feel confused sometimes in usage lol. 😂

Reply:
@luongvo I feel weird when we declare the PascalCase for Jetpack Compose components but leave it as snake_case in other classes. IMO. I think every class should have the same convention.

Reply:
@lydiasama Alright, it's in the `app` module and contributes to Compose UI logic so I made an update here https://github.com/nimblehq/android-templates/pull/367/commits/a0e3714a56943abb4611184bb1563c5399e7797f